far2l/src/cfg/config.cpp

```diff
@@ -388,6 +397,8 @@ static LONG_PTR WINAPI DirCfgDlgProc(HANDLE hDlg, int Msg, int Param1, LONG_PTR
 
 			dircfg_data->DirNameStyle = SendDlgMessage(hDlg, DM_LISTGETCURPOS, ID_DIRCFG_STYLE_COMBO, (LONG_PTR)0);
 			update_surrcombo( );
+			swprintf(tmp, 48, L"\"%ls\"", SymLinkNames[dircfg_data->DirNameStyle].CPtr());
```

Comment:
please use ARRAYSIZE(tmp) instead of hardcoded 48 (btw its 48 * 4)

Reply:
done